pkg/api/templates.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import "time"
+
+type TemplateRequest struct {
+	UUID            *string    `json:"uuid" readonly:"true" swaggerignore:"true"`
+	Name            *string    `json:"name"`                                        // Name of the template
+	Description     *string    `json:"description"`                                 // Description of the template
+	RepositoryUUIDS []string   `json:"repository_uuids"`                            // Repositories to add to the template
+	Arch            *string    `json:"arch"`                                        // Architecture of the template
+	Version         *string    `json:"version"`                                     // Version of the template
+	Date            *time.Time `json:"date"`                                        // Latest date to include snapshots for
+	OrgID           *string    `json:"org_id" readonly:"true" swaggerignore:"true"` // Organization ID of the owner
+}
+type TemplateResponse struct {
+	UUID        string
```

Comment:
not sure if it's related to what Stephen is seeing, but do you need `json:"uuid" ....` here?

Reply:
i'm not sure if it's related, but i should fix it either way!